recipes-ni/ni-hw-scripts/ni-hw-scripts-crio/init.d/handle_cpld_ip_reset

```diff
@@ -1,26 +1,9 @@
 #!/bin/sh
 
-# We are going to create a udev script to mount nirtfeatures under /dev
-# to make us more robust to changes in the absolute path under /sys. However
-# to allow things to continue to work until everything makes its way through
-# the system or in case that udev script doesn't run for some reason we fall
-# back to trying to find ip_reset in a likely spot.
-
 IP_RESET_FILE="/dev/nirtfeatures/ip_reset"
-SMBIOS_HELPER="/usr/share/nisysinfo/smbios_helper"
-
-if [ ! -e $IP_RESET_FILE ]; then
-        [ "${VERBOSE}" != "no" ] && echo "Unable to find ip_reset at normal spot. Attempting to find dynamically."
-        IP_RESET_FILE=`find /sys/bus/acpi/drivers/nirtfeatures -maxdepth 2 -follow -name ip_reset`
-fi
 
 IP_RESET_REQUESTED=""
-if [ -n "$IP_RESET_FILE" ]; then
-	grep -q '1' $IP_RESET_FILE && IP_RESET_REQUESTED="y"
-elif [ -r $SMBIOS_HELPER ]; then
-	source $SMBIOS_HELPER
-	get_ipreset && IP_RESET_REQUESTED="y"
```

Comment:
I remove the SMBIOS_HELPER fallback as well, because it does not work. get_ipreset actually expects an argument which is not provided by the call here.

Reply:
Right now if the udev rule is not there, then the script would fail with a file/directory not found error. Should we fail more gracefully?

Reply:
Yes? :)

Is it actually failure-case, if the udev rule is missing? I thought the raison d'etre for the initscript was to gracefully handle that case.

Reply:
Also; does the udev rule get installed by a package? Maybe this script should RECOMMEND that package.

Reply:
In the standup today, @gratian suggested that we should simplify the script if the udev rule is there and we can test on real hardware to validate that it does work. But having no fall-back does make me feel uneasy. I don't know enough about udev to know under what circumstance it would fail.

The udev rule is in `/etc/udev/rules.d/localextra.rules` which is from `udev-extraconf` package. Since the script would fail wihout the udev rule, maybe the relationship should be stronger than recommended?